### DISCUSSION TITLE
Trigger Mode: Root Note suggestion

### DISCUSSION THREAD

**Discussion by parrucasco** [bangcorrupt/hacktribe#338] [Ideas]
I've just come up with an idea of what i think is a more musical trigger mode, an option to play the "Selected in Key" note option. For example, if my pattern is set to D, when i press a pad will play that note.

this could be useful for playing single notes (bass, piano, synths) in tune with the song. the downside of course is any other sample that is not note-oriented, but it can be tuned to match the song (using Pitch/Glide knob for example)

i dont know if this implementation is diffult, i can imagine one way with my very basic knoledge of python but i'm as we say in Argentina "hablando sin saber" :D 

other ideas related for trigger mode:
. select note (jog whell)
. Key note only for [Instr] 
. Key note for individual part

**Comment by bangcorrupt** [comment]
This is related to #289.  Using the last played note would be useful in trigger and sequencer modes.
